report/thesis.tex

```diff
@@ -197,20 +218,39 @@ \chapter{Tendermint protocol overview and the fork accountability problem}
 
 - A process sends multiple PREVOTE/PRECOMMIT messages in a round *r* for different values (**equivocation**)
```

Comment:
If there are references to these scenarios, it would be a good idea to cite those references (or put them in footnotes).

Reply:
You can even have a full-fledged bibliography section ("References"), which usually looks very professional (but it's not necessary).

Reply:
Done, I added the bibliography and references for most of the material I read.